websocket_managed_conn_test.go

```diff
@@ -19,7 +19,7 @@ func TestRTMSingleConnect(t *testing.T) {
 	go testServer.Start()
 
 	// Setup and start the RTM.
-	slack.SLACK_API = testServer.GetAPIURL()
+	//slack.SLACK_API = testServer.GetAPIURL()
```

Comment:
There's something wonky with the imports in this `slack_test` package -- it wants to import itself via github.  But if I scratch that and import the slack package instead (which would be the local fork) it doesn't compile.  I commented the line off and it flags it as a conflict now.